app/views/partials/_surveyor_flashes.html.haml

```diff
@@ -0,0 +1,5 @@
+- types = flash.keys.map(&:to_sym).select { |k| [:notice, :error, :warning].include?(k) }
```

Comment:
Given that the flash’s keys are now strings, as you mention, why not eliminate the `.map(&:to_sym)` and check strings in the `select`? Or, perhaps even more succinctly, how about making the conditional:

```
if (flash.keys & ["notice", "error", "warning"]).any?
```

Reply:
I want to keep either a `to_sym` or `to_s` so that both 4.0 and 4.1 are supported for the transitionary period. But I do like the conciseness of what you have.

Reply:
And actually, `types` needs to be a symbol for the `flash_messages` lookup (we could change that, but for now, I'd rather keep the changes minimal).